src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+
+}
+
+@font-face {
```

Comment:
here you imported font, let us set them to `html` selector and use `inherit` everywhere else